### DISCUSSION TITLE
MAgPIE run finished! But Error in !is.null(selectedItems) && selectedItems == "###NOMATCH###" :   'length = 2' in coercion to 'logical(1)'

### DISCUSSION THREAD

**Discussion by DCCAISSON** [magpiemodel/magpie#706] [Q&A] [answered]
MAgPIE run finished!
Error in !is.null(selectedItems) && selectedItems == "###NOMATCH###" :
  'length = 2' in coercion to 'logical(1)'
Calls: <Anonymous> -> readGDX
Execution halted
2024-07-28 20:34:10.210933: unlocked model.

**Comment by tscheypidi** [comment]
hi @DCCAISSON,

could you share the `fulldata.gdx` of your run? This would help me to fix the problem.

**Reply by tscheypidi** [reply, marked as answer]
I could now reproduce the problem. It is caused by a bug in the gdx2-package which is now fixed in version 0.3.1 (https://github.com/pik-piam/gdx2). Please install version 0.3.1 or newer of this package and the problem should not occur anymore.
